tokens/usecase/pricing/chain/pricing_chain.go

```diff
@@ -255,3 +255,12 @@ func (c *chainPricing) computePrice(ctx context.Context, baseDenom string, quote
 func (c *chainPricing) InitializeCache(cache *cache.Cache) {
 	c.cache = cache
 }
+
+// Check this strategy allow falling back to another pricing source
```

Comment:
Recommending for consistency. Whenever we define a method spec on an interface, let's aim to refer to interface directly to avoid fragmenting the context and making it go out-of-date or sync
```suggestion
// ShouldFallback implements pricing.PricingSource
```